scss/_settings_grid.scss

```diff
@@ -5,15 +5,29 @@
 $grid-max-width: 80rem !default;
 $grid-columns-small: 4 !default;
 $grid-columns-medium: 6 !default;
+$grid-columns-medium-new: 4 !default;
```

Comment:
Please rename to denote it's 4 cols, like `grid-columns-medium-4` or something, I know it's weird, because it technically just aligns the variable name with it's value, but in the end that's what we are doing.

And `new` doesn't say anything, and will get "old" pretty quick.

Reply:
Renamed